lib/config/default.config.js

```diff
@@ -335,6 +335,13 @@ module.exports = {
               ttl: { type: 'keyword' },
               token: { type: 'keyword' }
             }
+          },
+          installations: {
+            dynamic: 'false',
+            properties: {
+              handler: { type: 'text' },
```

Comment:
Why storing the code of the function?

Reply:
I tought it would be useful to keep a record of what have been done but it may not be necessary

Reply:
It that case maybe a description of what this setup does is better